Added graphs for Module 3 exercises.

JackC92 opened the pull request:
Added graphs for Problem 8, 9, 10 of Module 3.

![image](https://user-images.githubusercontent.com/16367755/85373090-c0f30180-b500-11ea-978b-f1e57cc80fb4.png)
![image](https://user-images.githubusercontent.com/16367755/85373105-c5b7b580-b500-11ea-99aa-4cf2dfece3a4.png)
![image](https://user-images.githubusercontent.com/16367755/85373118-ce0ff080-b500-11ea-8a03-2f3c8e915dc9.png)
![image](https://user-images.githubusercontent.com/16367755/85373134-d536fe80-b500-11ea-9340-8e53e5ba07ff.png)
![image](https://user-images.githubusercontent.com/16367755/85373158-db2cdf80-b500-11ea-9d46-e837f1cce8d1.png)

siefkenj commented:
A couple of comments: can you label the figures for #8 on the graph itself? Perhaps make each one (S, T, T+S) a different color and make the labels matching colors.  The units on T+S are not uniform, can you make them uniform?

For #9, I like the idea of having new dots added in orange. Let's fill out all the orange dots that will fit on the grid. You can use a tikz loop for this, if you're daring enough.

JackC92 commented:
Updated the graphs based on the comments. I also changed a notation in Module 7 exercises, where `\vec n` and `\vec x` were used together to represent the normal vector.
![image](https://user-images.githubusercontent.com/16367755/85465376-bb380300-b576-11ea-9ff1-8c65eb4891bc.png)
![image](https://user-images.githubusercontent.com/16367755/85465410-c3903e00-b576-11ea-8c89-f8f402b35624.png)

siefkenj commented on book/modules/module3-exercises.tex:
```diff
@@ -245,6 +329,149 @@
+						\fill [WildStrawberry] (3,3) circle (2pt);
+					\end{axis}
+				\end{tikzpicture}
+				\item The set $(S+S)+S$ and $S+(S+S)$ are equal since vectors are associative. This means that $S+S+S$ is well-defined and we can drop the parentheses, so the expression $S+S+S$ makes sense.
```
-> `since vector addition is associative`

siefkenj commented on book/modules/module3-exercises.tex:
```diff
@@ -245,6 +329,149 @@
+					}
+				\end{tikzpicture}
+				
+				Notice that we obtain the orange points when we add additional $S$ to the set sum. Since we are considering the infinite sum $S+S+S+S+\cdots$, the process continues infinitely.
```
-> `add an additional $S$`

siefkenj commented on book/modules/module7-exercises.tex:
```diff
@@ -106,8 +106,8 @@
-						\vec x \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
-						\vec x \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
+						\vec n \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
+						\vec n \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
```
Put this back to `x`. Here `x` stands for `(x,y,z)` by convention.

JackC92 replied:
Should I also change the `n` in the line above to `x`?

siefkenj replied:
Ah, I didn't read the question carefully. Instead, let's keep these as `n`, but change the variables to `n_x`, `n_y`, and `n_z`.

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -447,7 +537,37 @@
+			\coordinate (A) at (2,1);
+			\coordinate (B) at (1,-1);
+			\draw [mygreen, thick, -] ($-0.5*(A)$) -- ($2*(A)$);
+			\draw [myorange, thick, -] ($(A)-3*(B)$) -- ($(A)+2*(B)$);
```
You shouldn't need the `-` as an argument to the `\draw` command.

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -324,8 +375,43 @@
-	
-		XXX Finish. Express answer both in vector form and as a set.
+		
+		The corresponding augmented matrix of the system is
```
of -> for

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -324,8 +375,43 @@
+			\end{bmatrix}
+		\]
+		
+		The matrix is already in pre-reduced row echelon form, so we need to zero above each pivot.
```
-> so we only need to

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -324,8 +375,43 @@
+			\end{bmatrix}
+		\]
+		
+		To solve the system
```
+ The third column is a free variable column, so we introduce ...

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -324,8 +375,43 @@
+		
+		The complete solution to the system in vector form is
+		\[
+			\mat{x\\y\\z} = \matc{-1-2t\\2+t\\t} = t\mat{-2\\1\\1}+\mat{-1\\2\\0}
```
-> `, and written as a set, the solution set is ...`

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -68,8 +93,34 @@
+			1 & 0 & 0 & 2\\
+			0 & 0 & 0 & 0\\
+			0 & 0 & 1 & 7
+			\end{bmatrix},\,
```
Use `\qquad` and no commas

siefkenj commented on book/modules/appendix2.tex:
```diff
@@ -68,8 +93,34 @@
+			\end{bmatrix}
+		\]
+		
+		$C$ is in reduced row echelon form and column 1, 2, 3 are the pivot columns. The fourth property fails for $A$. The first and second property fails for $B$. The second and third property fails for $D$.
```
Explain these in order, and in more detail (for example by highlighting in color the columns, etc that are incorrect). Keep in mind, this is the *very first* time students are seeing rref.

siefkenj commented on book/modules/module7-exercises.tex:
```diff
@@ -104,41 +104,41 @@
-						\vec x = t\mat{0\\-1\\1}.
+						\vec n = t\mat{0\\-1\\1}.
 					\]
 					This means that any non-zero multiple of $\mat{0&-1&1}$ is a normal vector for this plane, so the normal form of the plane is 
```
`the normal form` -> `a normal form`

siefkenj commented on book/modules/module7-exercises.tex:
```diff
@@ -104,41 +104,41 @@
 					\end{align*}
-					which has nullspace equal to $\Span{\mat{6\\7\\-6}}$.
+					which has nullspace equal to $\Span\Set*{\mat{6\\7\\-6}}$.
 					Since $\mat{0\\0\\9}$ is a point on the plane, we then get that the normal form of the plane is \[\mat{6\\7\\-6}\cdot \left(\mat{x\\y\\z}-\mat{0\\0\\9}\right)=0.\]
```
`the normal form` -> `a normal form`
